device_service.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+)
+
+type Device struct {
+	Id       int     `json:"id"`
+	Name     string  `json:"name"`
+	Interval int     `json:"interval"`
+	Value    float32 `json:"value"`
+}
+
+type DeviceService struct {
+	Database
+}
+
+func (service *DeviceService) createDevice(w http.ResponseWriter, r *http.Request) {
```

Comment:
This service is tied to http now. I would like to have a service that is clean from http. Please introduce a some intermediate http handler layer and provide to the service ready to use Device object.